src/controllers/motorcontroller.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, network_manager):
         super().__init__(network_manager)
         self.test_data = TestData()
```

Comment:
Could you also just add the path to the CSV file as an argument to the TestData() constructor? Something along the lines of `TestData("data/MotorControllerTestData.csv")`